packages/web/src/index.css

```
@@ -0,0 +1,7 @@
+@import 'tailwindcss';
+@plugin "daisyui" {
+  themes:
```

Biome (1.9.4)

[error] Unknown property is not allowed.

See CSS Specifications and browser specific properties for more details.
To resolve this issue, replace the unknown property with a valid CSS property.

(lint/correctness/noUnknownProperty)